raiden/transfer/events.py

```diff
@@ -214,6 +244,30 @@ class EventInvalidReceivedUnlock(Event):
     reason: str
 
 
+@dataclass
+class EventInvalidReceivedWithdrawRequest(Event):
+    """ Event emitted when an invalid withdraw request is received. """
+
+    total_withdraw: WithdrawAmount
```

Comment:
semantics of `total_withdraw` are a little vague here (and in the other `*Invalid*` below): since it is an error, I am not sure if `total` is the legal, or the invalid. If refering to the invalid, how about
```
attempted_withdraw
```
?